duration_fuzz.go

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build gofuzz
 // +build gofuzz
```

Comment:
What's with both of these? Seems redundant...

Reply:
See the 1.18 release notes, they're migrating from one syntax to the other, keeping both for now for compat. This is from running gofmt, I didn't add these by hand

Reply:
Oh! TIL. Very cool.

I installed Go 1.18 today but haven't done much with it yet.